ModelCatalogueGenomicsPlugin/src/groovy/org/modelcatalogue/gel/export/CancerTypesJsonExporter.groovy

```diff
@@ -97,6 +100,49 @@ class CancerTypesJsonExporter {
         }
     }
 
+
+    private void buildSubTypeOrPresentation(Map map, String subTag, CatalogueElement model) {
+        if ( SUB_TYPE.equals(subTag) ) {
+            def subTypes = []
+            map.put(ID_TAG, "$model.id")
+            map.put(SUB_TYPE, model.name)
+            map.put(SUB_TYPES, subTypes)
+            iterateEnumTypes(model, subTypes)
+        }
+        if ( PRESENTATION.equals(subTag) ) {
+            map.put(subTag, model.name)
+            map.put(DESCRIPTION_TAG, model.description ?: '')
+        }
+    }
+
+    private void iterateEnumTypes(CatalogueElement model, List subtypes) {
+
+        if(model instanceof DataElement && model.dataType instanceof EnumeratedType) {
```

Comment:
sadly, this is not that easy 😞 Hibernate + supertypes in relations just sucks. only error proof way I've found so far is `HibernateHelper.getClass(model.dataType) == EnumeratedType` otherwise this will fail occasionally and not be recognize. Even the recommended `model.dataType.instanceOf(EnumeratedType)` seems not to be working 100%.